plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -259,13 +290,17 @@ func (e *Elasticsearch) gatherNodeStats(url string, acc telegraf.Accumulator) er
 
 		now := time.Now()
 		for p, s := range stats {
+			if !e.individualStatIncluded(p) {
+				continue
+			}
 			f := jsonparser.JSONFlattener{}
 			// parse Json, ignoring strings and bools
 			err := f.FlattenJSON("", s)
 			if err != nil {
 				return err
 			}
-			acc.AddFields("elasticsearch_"+p, f.Fields, tags, now)
+			measurement := "elasticsearch_" + p
+			acc.AddFields(measurement, f.Fields, tags, now)
```

Comment:
Revert this hunk, it's just a style change and no reason to make history more complex.

Reply:
Agreed, it is a remainder of something I tried earlier.